packages/amplication-client/src/VersionControl/Build.tsx

```diff
@@ -32,8 +31,6 @@ const Build = ({ build, onError, open }: Props) => {
       <PanelHeader>Build details</PanelHeader>
       <div className={`${CLASS_NAME}__message`}>{data.build.message}</div>
       <UserAndTime account={account} time={build.createdAt} />
-      <BuildHeader build={data.build} deployments={data.build.deployments} />
```

Comment:
you remove entirly <BuildHeader> and I am not sure if it is still being used... below I can see that you also updated <BuildHeader> so what did you mean to do?